docker/testnet/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:20.04
```

Comment:
we don't really need testnet anymore, but this structure is still useful for when we have multiple chains (turing, oak, etc)